### PR TITLE
Ecs compares jarvis

### PR DESCRIPTION
**Pull request by farque65**

### CODE REVIEW

File: src/lambda_function.py [hunk 1]
```diff
@@ -109,23 +124,72 @@ def post_to_slack(val):
...
+			print "ephemeral_message_request error "+str(e)
+			if ephemeral_message_request.status_code != 200:
+				raise ValueError(
+					'In send_to_slack ephemeral Slack returned status code %s, the response text is %s' % (
+						ephemeral_message_request.status_code, ephemeral_message_request.text)
+				)
+
+		# after sending a message to your currenet channel,
+		#  then send another to the desired slack channel
+
```
Comment (haggaret) [inline src/lambda_function.py:175]: Sweet!

File: src/plugins/ecs/__init__.py [hunk 2]
```diff
@@ -452,7 +462,7 @@ def information():
 	jarvis ecs describe|desc <cluster> [in <region/account>]
 	jarvis ecs describe|desc <service> <cluster> [in <region/account>]
 	jarvis ecs list tasks[---<task_name_optional>] running <cluster> [in <region/account>]
-	jarvis ecs compare <cluster> [in <region> <account>] with <cluster> [in <region> <account>]"""
+	jarvis ecs compare [<cluster>] into <region> <account> with [<cluster>] into <region> <account>"""
```
Comment (haggaret) [inline src/plugins/ecs/__init__.py:465]: into instead of in?

**Comment by haggaret** [comment]
Looks good.

File: src/plugins/ecs/__init__.py (hunk 2 above)
Comment (farque65) [reply src/plugins/ecs/__init__.py:465]: yes I found that using "in" would conflict with the other commands in jarvis, it leads to jarvis taking in the last region from one of the comparables. Using something besides "in" sidesteps jarvis parsing out the two comparables

**Pull request merged**